### PR TITLE
modified POST API for creating arrangement

### PR DESCRIPTION
**Pull request by Jeffchiucp**
- modified POST API for creating arrangement #19

### CODE REVIEW

**Comment by Jeffchiucp** [comment]
test the arrangement object screen shot 2018-12-18 at 11 32 48 pm

File: app.py [hunk 1]
```diff
@@ -43,10 +43,10 @@ def default(self, o):
         return json.JSONEncoder.default(self, o)
 
 
-@app.route("/")
+@app.route("/",methods=['GET'])
 def home_page():
     access_token = session.get('access_token')
-    if access_token is None:
+    if access_token  is None:
```
Comment (deonjo) [inline app.py]: get rid of extra space

File: app.py [hunk 2]
```diff
@@ -93,10 +86,23 @@ def get_access_token():
 
 @app.route('/arrangement', methods=['POST'])
 @app.route('/api/v1/arrangement', methods=['POST'])
+
```
Comment (deonjo) [inline app.py]: can u get rid of this space

File: app.py [hunk 3]
```diff
@@ -93,10 +86,23 @@ def get_access_token():
...
     arrangement = request.json
     json_data = validate_arrangement(arrangement)
-    if json_data == True:
+
+    google_id = arrangement.get("user")
+
+    sync_users = []
+    if 'users' in arrangement:
+        for user in arrangement['users']:
+            obj_user = {"user": user, "google_id": google_id }
```
Comment (deonjo) [inline app.py]: i don't get why are you adding this google_id to the users?

File: app.py [hunk 4]
```diff
@@ -106,29 +112,35 @@ def save_arrangement():
         else:
             mdb.add_arrangement(data)
```
Comment (deonjo) [inline app.py]: I think you need to find the old arrangement if the arrangement already exists in mdb

File: app.py [hunk 5]
```diff
@@ -106,29 +112,35 @@ def save_arrangement():
...
         return jsonify({'message':'json is not validate'})
 
 
+
 @app.route('/arrangement', methods=['GET'])
 @app.route('/api/v1/arrangement', methods=['GET'])
 @app.route('/arrangement/<string:arrangement_id>', methods=['GET'])
 @app.route('/api/v1/arrangement/<string:arrangement_id>', methods=['GET'])
 def get_arrangement(arrangement_id=None):
+
```
Comment (deonjo) [inline app.py]: get rid of space

File: app.py [hunk 6]
```diff
@@ -106,29 +112,35 @@ def save_arrangement():
...
     if arrangement_id is None:
-        return JSONEncoder().encode(mdb.get_all_arrangements())
+
```
Comment (deonjo) [inline app.py]: get rid of space

File: app.py (hunk 3 above)
Comment (deonjo) [reply app.py]: the user should be identified by a SINGLE ID. not a combination of id.

**Comment by deonjo** [comment]
I think there's a conflict in this one now.

**Comment by Jeffchiucp** [comment]
resolved the conflict and merge master branch

**Comment by Jeffchiucp** [comment]
i think this branch is similar to master, so there's no need to keep it open.

**Pull request closed without merging**